pkgs/development/python-modules/hatch-regex-commit/default.nix

```diff
@@ -3,34 +3,32 @@
   buildPythonPackage,
   fetchFromGitHub,
   hatchling,
-  pythonOlder,
+  pytest-cov-stub,
 }:
 
 buildPythonPackage rec {
   pname = "hatch-regex-commit";
-  version = "0.0.3";
+  version = "0.0.4";
   pyproject = true;
 
-  disabled = pythonOlder "3.7";
-
   src = fetchFromGitHub {
     owner = "frankie567";
     repo = "hatch-regex-commit";
     tag = "v${version}";
-    hash = "sha256-E0DIBBaDmTCsZQ41NcjcbzgJ16BwhdexlrGWBdf77oA=";
+    hash = "sha256-xdt3qszigdCudt2+EpUZPkJzL+XQ6TnVEAMm0sV3zwY=";
   };
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace-fail 'dynamic = ["version"]' 'version = "${version}"' \
-      --replace-fail ', "hatch-regex-commit"' "" \
-      --replace-fail " --cov-report=term-missing --cov-config=pyproject.toml --cov=hatch_regex_commit --cov=tests" ""
+      --replace-fail 'dynamic = ["version"]' 'version = "${version}"'
   '';
 
   build-system = [ hatchling ];
 
   dependencies = [ hatchling ];
 
+  nativeCheckInputs = [ pytest-cov-stub ];
```

Comment:
I just realized we're not even running the tests, so this can probably be removed.